os/docker/gno/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# This dockerfile will be used to bootstrap the docker changes for gno, and will be built hourly.
+
+FROM ghcr.io/faddat/archlinux
+
+ENV GOPATH=/go
+
+# everything needed for gno development
+# fully supports all databases and their compression schemes, with nothing additional.
+RUN pacman -Syyu --noconfirm go base-devel git leveldb rocksdb make snappy lz4 protobuf
+
+# Useful tools
+RUN go install mvdan.cc/gofumpt@latest && \
```

Comment:
having less dependencies is a thing that fits more with the current project's spirit & style

however, my main concern right now is to get more feedback about the goal of this PR, who will use it etc

I've no doubt that this PR brings value, I personally don't know yet how much it's important to be done now, who it will be useful for, and if it will add additional maintenance

Reply:
ah, so the main things that it is useful for is giving gno a known-good runtime environment that runs from embedded (rpi images can auto build from these containers) to the cloud (vms too) to container systems (its a container so plays well in kube and the like) to bare metal servers.

Reply:
Let me make a smaller pull request, and walk you through a developer implication of these changes.